app/services/weather_map_service.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class WeatherMapService
+  attr_accessor :uri
+
+  def initialize
+    return if Rails.application.credentials.openweathermap.blank?
+
+    lat = Rails.application.credentials.openweathermap[:lat]
+    lon = Rails.application.credentials.openweathermap[:lon]
+    api_key = Rails.application.credentials.openweathermap[:api_key]
+    base_url = "http://api.openweathermap.org"
+
+    @uri = "#{base_url}/data/2.5/onecall?lat=#{lat}&lon=#{lon}&appid=#{api_key}&units=metric&exclude=minutely&lang=de"
```

Comment:
muss die `2.5` mit der Zeit aktualisiert werden? wollen wir die version dann vllt an einer anderen stelle haben, um das dann nicht im code machen zu müssen. vllt auch credentials.
oder als konstante im code, um die stelle klarer zu finden in der zukunft?

Reply:
braucht vllt. später mal eine bestimmte app version eine andere bestimmte open weather map version :o 
hoffentlich nich

Reply:
ich würde jetzt erstmal vermuten, dass die OWM versionen hinreichend lang supported werden, dass wir uns wegen alter app versionen keine gedanken machen müssen.
wenn wir dann eine neue version serverseitig nutzen wollen, dann können wir zu dem zeitpunkt versionierung mit dieser version als fallback einbauen, sodass alte apps weiterlaufen.
ich sehe erstmal keine notwendigkeit da aufwand drauf zu verwenden, wenn @marcometz nicht der meinung ist, dass ein solcher fallback später zu viel mehraufwand bedeutet.